kura/org.eclipse.kura.event.publisher/OSGI-INF/metatype/org.eclipse.kura.event.publisher.EventPublisher.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022 Eurotech and/or its affiliates. All rights reserved.
+-->
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.event.publisher.EventPublisher" 
```

Comment:
As I was writing in the previous comment, maybe it makes sense to have another factory for the log event? So that the fields are already pre-filled for configuration events?